### PR TITLE
Update minecraft to use new installer and other improvements

### PR DESCRIPTION
**Pull request by Jacalz**
# Description
This Pull Request modifies the Minecraft install instructions to use the new launcher from version 1.13.0, that is official on Linux from that release and onward. The new launcher is slightly more complicated to set up but it is maintained and has a lot better user interface.

This PR also updates the image to use a high resolution screenshot with the default texture pack instead of Faithful HD and I added a section to help out Mate users when they create the desktop entry.

### CODE REVIEW

File: gaming/minecraft/en.md [hunk 1]
```diff
@@ -33,7 +44,7 @@ Type=Application
 Name=Minecraft
 Comment=Minecraft Launcher
 Icon=minecraft
-Exec=java -jar Minecraft.jar
+Exec=/opt/minecraft/./minecraft-launcher.sh
```
Comment (JoshStrobl) [inline gaming/minecraft/en.md]: `./` isn't needed. Can just be `/opt/minecraft/minecraft-launcher.sh`

**Comment by Jacalz** [comment]
Haha, I just went with the same size as the old one because I was afraid of making it too big 😬 Won’t be able to get a new screenshot until Monday though...

File: gaming/minecraft/en.md (hunk 1 above)
Comment (Jacalz) [reply gaming/minecraft/en.md]: Good to know, will fix 👍

**Comment by Jacalz** [comment]
This has now been updated to remove "./" from the desktop entry and there is a much better looking 1080p screenshot instead of the old crappy one. Commits have also been squashed. @JoshStrobl

**Comment by Jacalz** [comment]
In other words, this is ready for a re-review :)

**Comment by Jacalz** [comment]
I did not take in to account that we are one hour more forward in the summer in Sweden so I changed it to +02:00 now. Would you mind taking a look at this now since it is ready? @JoshStrobl

File: gaming/minecraft/en.md [hunk 2]
```diff
@@ -10,19 +10,30 @@ lastmod = "2017-05-19T13:19:32+03:00"
...
-Now we can create a desktop icon for it:
+Extract files and remove old archive:
+``` bash
+sudo tar xvf /opt/minecraft/Minecraft.tar.gz --strip=1 -C /opt/minecraft/
+sudo rm /opt/minecraft/Minecraft.tar.gz
+```
+
+#### Now we can create a desktop icon for it:
 
+On Solus Budgie or Gnome:
```
Comment (JoshStrobl) [inline gaming/minecraft/en.md]: GNOME

File: gaming/minecraft/en.md [hunk 3]
```diff
@@ -10,19 +10,30 @@ lastmod = "2017-05-19T13:19:32+03:00"
...
 ``` bash
 sudo gedit /usr/share/applications/minecraft.desktop
 ```
+On Solus Mate:
```
Comment (JoshStrobl) [inline gaming/minecraft/en.md]: MATE

Comment (Jacalz) [reply gaming/minecraft/en.md]: Done with https://github.com/Jacalz/help-center-docs/commit/ae73cd9bfe524fe9152100f09ef04666b41e34f7

File: gaming/minecraft/en.md (hunk 2 above)
Comment (Jacalz) [reply gaming/minecraft/en.md]: Done with 
https://github.com/Jacalz/help-center-docs/commit/ae73cd9bfe524fe9152100f09ef04666b41e34f7

**Comment by Jacalz** [comment]
I have pushed the changes and this should thus be ready for a re-review

**Comment by JoshStrobl** [comment]
LGTM, thanks!

**Pull request merged**